testy/tagged.go

```diff
@@ -0,0 +1,96 @@
+package testy
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	TagUnit        = "unit"
+	TagIntegration = "integration"
+	TagPostgres    = "postgres"
+	TagRabbit      = "rabbit"
+)
+
+// TaggedTestsEnvVar defines the name of the environment variable for the tagged tests.
+// Example:
+// 		TaggedTestsEnvVar="TEST_TAGS"
+//		env TEST_TAGS="unit" go test ./...
+var TaggedTestsEnvVar = "TEST_TAGS"
+
+// TaggedTest runs the test if the provided tag matches at least one runtime tag.
+// Example:
+// 		func TestSomething(t *testing.T) {
+//    		TaggedTest(t, "unit")
+//    		...
+// 		}
+// Run with:
+// 		env TEST_TAGS="unit,integration" go test ./...
+func TaggedTest(t *testing.T, testTag string) {
+	tags := getRuntimeTags()
+	if shouldRun := tags.empty() || tags.contains(testTag); !shouldRun {
+		t.SkipNow()
```

Comment:
I think we should use `t.skipf(...)` and print the accepted test tags, it is important that this functionality is discoverable